### DISCUSSION TITLE
Feature request: make the UI minimize on ESC

### DISCUSSION THREAD

**Discussion by mhesham32** [jotaijs/jotai-devtools#108] [Ideas]
it will be nice to be able to close/minimize the UI of the dev tools be pressing ESC key instead of using the mouse to click on the minimize icon

**Comment by arjunvegda** [comment]
Hey @mhesham32 

That's a good idea! I wonder if we could provide ways for consumers to implement this on their end (or configure it somehow via props or something). I don't know how I feel about including it by default as it could conflict with user's apps.

**Reply by mhesham32** [reply]
Hi @arjunvegda ,
thanks for your reply, I personally don't see it conflicting with user's apps as long as the UI is focused, but surly it should be customizable via Props
